test/README.md

```diff
@@ -0,0 +1,63 @@
+[![Build Status](https://travis-ci.org/webrtc/samples.svg)](https://travis-ci.org/webrtc/samples)
+
+# Intro #
+Selenium webdriver, node, testling and travis-multirunner is used as the testing framework. Selenium webdriver drives the browser, node and Testling manages the tests while travis-multirunner dowloads and installs the browsers to be tested on, e.g creates the testing matrix.
```

Comment:
Selenium WebDriver, Node, Testling and travis-multirunner are used as the testing framework. Selenium WebDriver drives the browser; Node and Testling manage the tests, while travis-multirunner downloads and installs the browsers to be tested on, i.e. creates the testing matrix.